packages/ui/cart/drawer/Upsell.tsx

```diff
@@ -42,12 +42,14 @@ export function Upsell({ item }: UpsellProps) {
             borderColor="brand.200"
             borderStyle="solid"
          >
-            <Text color="brand.500" fontWeight="semibold">
+            <Link href={item.url} color="brand.500" fontWeight="semibold">
                {item.marketingTitle ?? item.name}
-            </Text>
+            </Link>
             <Text color="brand.500">Add the product below to your order</Text>
             <HStack mt="3" align="center" spacing="3">
-               <CartLineItemImage src={item.imageSrc} alt={item.name} />
+               <a href={item.url}>
+                  <CartLineItemImage src={item.imageSrc} alt={item.name} />
+               </a>
                <Text fontSize="sm">{item.marketingBlurb}</Text>
             </HStack>
```

Comment:
Opinion: I know the spec only requests wrapping the image and the title, but I personally would expect the entire upsell card to be a link, since this is what we do on product list pages, featured product section, and the cross sell section.

Reply:
@sterlinghirsh what do you think of this design idea?